backend/tournesol/tests/test_api_statistics.py

```diff
@@ -0,0 +1,133 @@
+        )
+        self._list_of_users = [user_1, user_2]
+
+        video_1 = VideoFactory(
+            uploader="uploader1",
+            rating_n_ratings=2,
+        )
+        video_2 = VideoFactory(
+            uploader="uploader2",
+            rating_n_ratings=3,
+        )
+        video_3 = VideoFactory(
+            uploader="uploader2",
+            rating_n_ratings=4,
+        )
+        
+        Video.objects.filter(pk=video_1.pk).update(add_time=timezone.now() - timedelta(days=5))
+        Video.objects.filter(pk=video_2.pk).update(add_time=timezone.now() - timedelta(days=29))
+        Video.objects.filter(pk=video_3.pk).update(add_time=timezone.now() - timedelta(days=60))
+
+        self._list_of_videos = [video_1, video_2, video_3]
+
+        comparison_1 = ComparisonFactory(
+                user=user_1, video_1=video_1, video_2=video_2,
+                duration_ms=102
+        )
+        comparison_2 = ComparisonFactory(
+                user=user_2, video_1=video_1, video_2=video_3,
+                duration_ms=104
+        )
+        comparison_3 = ComparisonFactory(
+                user=user_2, video_1=video_2, video_2=video_3,
+                duration_ms=302
+        )
+
+        Comparison.objects.filter(pk=comparison_1.pk).update(datetime_lastedit=timezone.now() - timedelta(days=5))
+        Comparison.objects.filter(pk=comparison_2.pk).update(datetime_lastedit=timezone.now() - timedelta(days=29))
+        Comparison.objects.filter(pk=comparison_3.pk).update(datetime_lastedit=timezone.now() - timedelta(days=60))
+            
+        self._list_of_comparisons = [comparison_1, comparison_2, comparison_3]
+       
+
+    def test_video_count_is_right(self):
+        """
+        Test if the number of video is right
+        """
+
+        client = APIClient()
+
+        response = client.get(
+            reverse("tournesol:statistics_detail"),
+            format="json",
+        )
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+        video_count = response.data["video_count"]
+        last_month_video_count = response.data["last_month_video_count"]
+
+        self.assertEqual(video_count, len(self._list_of_videos))
+        self.assertEqual(last_month_video_count, 2)
+
+    def test_user_count_is_right(self):
+        """
+        Test if the number of user is right
+        """
+
+        client = APIClient()
+
+        response = client.get(
+            reverse("tournesol:statistics_detail"),
+            format="json",
+        )
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+        user_count = response.data["user_count"]
+        last_month_user_count = response.data["last_month_user_count"]
+
+        self.assertEqual(user_count, len(self._list_of_users))
+        self.assertEqual(last_month_user_count, 1)
+
+    def test_video_comparison_is_right(self):
+        """
+        Test if the number of comparison is right
+        """
+
```

Comment:
I'd like to propose to reword this a bit. The formulation `is_right` doesn't seem to be needed here, as there no test that will ensure something `is_wrong`.

How about something like this?

```suggestion
    def test_comparison_stats(self):
    """"
    An anonymous user can get statistics about comparisons.
    """"
```

Reply:
Ok I made the change.